rocketpy/Rocket.py

```diff
@@ -13,8 +13,15 @@
 
 from .Function import Function
 from .Parachute import Parachute
-from .AeroSurfaces import AeroSurfaces
-from .AeroSurfaces import NoseCone, TrapezoidalFins, EllipticalFins, Tail
+from .AeroSurfaces import (
+    AeroSurfaces,
```

Comment:
AeroSurfaces currently not being used in the rest of the code.

Error in code or should we just remove this import?